modules/tinymce/src/core/test/ts/browser/dom/SelectionQuirksTest.ts

```diff
@@ -64,8 +65,20 @@ describe('browser.tinymce.core.dom.SelectionQuirksTest', () => {
     TinySelections.setSelection(editor, [ 0, 0, 0 ], 0, [ 0, 0 ], 0);
     TinyContentActions.keyup(editor, Keys.left(), { shift: true });
     assertNormalizeCounter(0);
-    TinyContentActions.keyup(editor, 17, { }); // single ctrl
-    assertNormalizeCounter(1);
+    const isMac = Env.os.isMacOS() || Env.os.isiOS();
+    editor.dispatch('keyup', new KeyboardEvent('keyup', { key: 'Control' }));
+    assertNormalizeCounter(isMac ? 1 : 0);
```

Comment:
Why is this different on Mac? The goal should be to have it be consistent across browsers.

Reply:
This change was removed. Should this discussion be resolved ?